projects/schematics/src/migrations/4_0/component-deprecations/data/outlet.directive.migration.ts

```diff
@@ -0,0 +1,14 @@
+import { OUTLET_DIRECTIVE } from '../../../../shared/constants';
+import { ComponentData } from '../../../../shared/utils/file-utils';
+
+//projects/storefrontlib/src/cms-structure/outlet/outlet.directive.ts
+export const OUTLET_DIRECTIVE_MIGRATION: ComponentData = {
```

Comment:
I know I said it would be a component-deprecations, but I didn't mean deprecating the outlet.directive.

Reply:
After double checking, I believe it falls under config-deprecation, but follow-up with Kris as the issue asks to remove the property `context` from OutletContextData. 

I believe you'll need to update references that uses .context only, but again follow-up with Kris on this